### DISCUSSION TITLE
Uc without system webview?

### DISCUSSION THREAD

**Discussion by ghost** [ungoogled-software/ungoogled-chromium-android#69] [Q&A] [answered]
@wchen

Hi 
so ungoogle chromium doesn't need android  system webview? Can you confirm this? I disable android system webview and UC Still Working While Other Browser Like DDG For Example Are Crash

**Comment by wchen342** [comment, marked as answer]
The browser and webview are two separate things, unless you are using `TriChrome` in which you cannot disable the library.

**Reply by ghost** [reply]
Which mean ungoogle chromium not obliged to use the android webview system right? So i can disable them happily YAY! 🤣

**Reply by wchen342** [reply]
Yes you can use them separately.

**Reply by ghost** [reply]
> Yes you can use them separately.

I have to admit it 
You are great developer ever!
How do you know that i am hate Android System webview? (Yes default Asw are annoying because developed by google and i don't want to use it)

**Reply by wchen342** [reply]
You can use [this](https://forum.xda-developers.com/t/magisk-module-webview-switcher.3936964/) to install UC/Bromite webview if you have magisk.

**Reply by ghost** [reply]
> You can use [this](https://forum.xda-developers.com/t/magisk-module-webview-switcher.3936964/) to install UC/Bromite webview if you have magisk.

Well the only issues newer phone is hard to root 
And have high risk too 
So i can't replace System Webview
